config/generic-mks-sgenl-v2.cfg

```diff
@@ -0,0 +1,216 @@
+[tmc2209 stepper_y]
+uart_pin: P1.8
+run_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##--------------------------------------------------------------------
+#Z Stepper Settings
+##--------------------------------------------------------------------
+
+## Z0 Stepper
+[stepper_z]
+step_pin: P0.22
+dir_pin: P2.11
+enable_pin: !P0.21
+rotation_distance: 8
+full_steps_per_rotation:400
+microsteps: 16
+endstop_pin: ^!P1.28
+position_endstop: 0
+position_min: -1.3
+position_max: 200
+homing_positive_dir: false
+homing_speed: 6
+second_homing_speed: 4
+homing_retract_dist: 3
+
+## Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z]
+uart_pin: P1.10
+run_current: 0.7
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+## Z1 Stepper
+[stepper_z1]
+step_pin: P1.9
+dir_pin: P1.14
+enable_pin: !P0.10
+rotation_distance: 8
+full_steps_per_rotation:400
+microsteps: 16
+
+[tmc2209 stepper_z1]
+uart_pin: P1.17
+run_current: 0.1
+sense_resistor: 0.110
+stealthchop_threshold: 0
+#####################################################################
+#Extruder
+#####################################################################
+[extruder]
+step_pin: P2.13
+dir_pin: !P0.11
+enable_pin: !P2.12
+## Update value below when you perform extruder calibration
+## If you ask for 100mm of filament, but in reality it is 98mm:
+## rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+rotation_distance: 23.46041055718
+max_extrude_only_distance: 110.0
+microsteps: 16
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: P2.7
+## Validate the following thermistor type to make sure it is correct
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: P0.23
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+control = pid
+pid_kp = 20.741
+pid_ki = 0.981
+pid_kd = 109.667
+
+
+[tmc2209 extruder]
+uart_pin: P1.15
+run_current: 0.3
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# Bed Heater
+#####################################################################
+
+[heater_bed]
+# SSR Pin - BED_OUT
+heater_pin: P2.5
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: P0.24
+control: watermark
+min_temp: 0
+max_temp: 110
+
+
+#####################################################################
+# Probe
+#####################################################################
+
+#[probe]
+#pin: ^!P1.25
+#x_offset: -10
+#y_offset: 0
+#z_offset: 1
+#speed: 15.0
+#lift_speed: 30.0
+#samples: 3
+#samples_result: average
+#sample_retract_dist: 2.0
+#samples_tolerance: 0.050
+#samples_tolerance_retries: 8
+#####################################################################
+# Fan Control
+#####################################################################
+
+[fan]
+pin: P2.4
+off_below: 0.10
+
+[heater_fan hotend_fan]
+pin: P1.4
+max_power: 1.0
+fan_speed: 1.0
```

Comment:
Not needed. Default anyway

Reply:
i thought both values are required for klipper to compile?

Reply:
These are the default values, so no need to set them.